### ISSUE TITLE
Plot wont be able to auto resize to fit the view from latest version

### ISSUE THREAD

**Issue by JefferyChiang** [gradio-app/gradio#1464]
### Describe the bug

I use Plot demo from docs.
It can auto fit the size in gradio 3.0.1, but not be able to do so in gradio 3.0.11.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction
blocks_kinematics plot demo code from docs
```
import matplotlib
matplotlib.use('Agg')
import matplotlib.pyplot as plt
import numpy as np

import gradio as gr


def plot(v, a):
    g = 9.81
    theta = a / 180 * 3.14
    tmax = ((2 * v) * np.sin(theta)) / g
    timemat = tmax * np.linspace(0, 1, 40)[:, None]

    x = (v * timemat) * np.cos(theta)
    y = ((v * timemat) * np.sin(theta)) - ((0.5 * g) * (timemat**2))

    fig = plt.figure()
    plt.scatter(x=x, y=y, marker=".")
    plt.xlim(0, 100)
    plt.ylim(0, 60)
    return fig


demo = gr.Blocks()

with demo:
    gr.Markdown(
        "Let's do some kinematics! Choose the speed and angle to see the trajectory."
    )

    with gr.Row():
        speed = gr.Slider(1, 30, 25, label="Speed")
        angle = gr.Slider(0, 90, 45, label="Angle")
    output = gr.Plot()
    btn = gr.Button("Run")
    btn.click(plot, [speed, angle], output)

demo.launch()
```

### Screenshot

from gradio 3.0.1
![image](https://user-images.githubusercontent.com/36859596/172130723-c68bb190-90c7-465a-be07-6eda654ef16a.png)

from gradio 3.0.11
![image](https://user-images.githubusercontent.com/36859596/172130872-e83f3bf5-8b7d-46d3-9ea0-f09421e78fc7.png)

### Logs

```shell
None
```

### System Info

```shell
win10
python 3.7.7
gradio 3.0.11
chrome 101.0.4951.67
```

### Severity

annoying

**Comment by dawoodkhan82** [comment]
@abidlabs @JefferyChiang I don't notice this issue when running the demo. Can you confirm it's been resolved?

**Comment by abidlabs** [comment]
Yes, it was fixed by a user contribution, #1550!
